cardano-tracer/test/Cardano/Tracer/Test/Forwarder.hs

```diff
@@ -190,8 +190,9 @@ doConnectToAcceptor TestSetup{..} snocket muxBearer address timeLimits (ekgConfi
 
   forwarderApp
     :: [(RunMiniProtocol 'Mux.InitiatorMode initCtx respCtx LBS.ByteString IO () Void, Word16)]
+    -> ForwardingVersionData
     -> OuroborosApplication 'Mux.InitiatorMode initCtx respCtx LBS.ByteString IO () Void
-  forwarderApp protocols =
+  forwarderApp protocols _ =
```

Comment:
There's no motivation to change type signature of `forwarderApp`. Can that be reverted?